### DISCUSSION TITLE
I need more explanation for use this in IONIC 6

### DISCUSSION THREAD

**Discussion by Adrinc** [admob-plus/admob-plus#517] [Q&A]
I´m building an Ionic 6 cordova app. And I find really hard to make this plugin works. I need help to clarify the correct way to make this works:
1.-What is the diference between the installation provided from ionic https://ionicframework.com/docs/native/admob-plus
     and the installation provided by admbob-plus documentation https://admob-plus.github.io/docs/ionic/installation

----- Ionic docs ---
ionic cordova plugin add cordova-admob-plus 
npm install @awesome-cordova-plugins/admob-plus 

----- Admob-plus docs----
ionic cordova plugin add admob-plus-cordova
npm install @admob-plus/ionic

2.-which one should I use for my ionic 6 project or I need to install both?
3.- In Admob plus docs you find the way to add the plugin is simple modify the packed.json to add your APP_ID and then 
"ionic cordova plugin add admob-plus-cordova". But I have facing the problem if you do in this way your android manifes will still add ca-app-pub-xxx~yyy generating duplicate ID (Error).
So is correct if insted of that I use ionic cordova plugin add admob-plus-cordova --save --variable APP_ID_ANDROID="MyID"?

4.- Do I need to do some other missing step in admob-plus docs? because the simple fact to add and install this plugin from admob-plus docs and importe it into my app.module.ts make the app freezes after splash screen.

Any help will be really appreciate

**Comment by ratson** [comment]
@Adrinc what is the output of `npx admob-plus doctor`?

You may find the following useful,
https://admob-plus.github.io/docs/troubleshooting/
https://github.com/admob-plus/admob-plus/tree/master/examples/ionic-angular

**Comment by TyeS2K** [comment]
I am also experiencing an issue with APP_ID. I followed the Google Admob docs and included the "APPLICATION_ID" in the manifest. I noticed it wasn't copied in the package.json, where there is a variable: APP_ID_ANDROID.

I am using volt.builder to building the cordova app, and was wondering if there was a reason for the change in name, and why the values are mirror'ed. Any help would be appreciated.
